pkg/planner/core/plan_clone_generated.go

```diff
@@ -222,11 +222,83 @@ func (op *PhysicalIndexReader) CloneForPlanCache(newCtx base.PlanContext) (base.
 	}
 	cloned.indexPlan = indexPlan.(base.PhysicalPlan)
 	cloned.IndexPlans = flattenPushDownPlan(cloned.indexPlan)
-	cloned.OutputColumns = util.CloneCols(op.OutputColumns)
+	cloned.OutputColumns = util.CloneColumns(op.OutputColumns)
 	cloned.PlanPartInfo = op.PlanPartInfo.Clone()
 	return cloned, true
 }
 
+// CloneForPlanCache implements the base.Plan interface.
+func (op *PointGetPlan) CloneForPlanCache(newCtx base.PlanContext) (base.Plan, bool) {
+	cloned := new(PointGetPlan)
+	*cloned = *op
+	cloned.Plan = *op.Plan.CloneWithNewCtx(newCtx)
+	cloned.schema = op.schema.Clone()
+	if op.PartitionIdx != nil {
+		cloned.PartitionIdx = new(int)
+		*cloned.PartitionIdx = *op.PartitionIdx
+	}
+	if op.Handle != nil {
+		cloned.Handle = op.Handle.Copy()
+	}
+	if op.HandleConstant != nil {
+		cloned.HandleConstant = op.HandleConstant.Clone().(*expression.Constant)
+	}
+	cloned.IndexValues = util.CloneDatums(op.IndexValues)
+	cloned.IndexConstants = util.CloneConstants(op.IndexConstants)
+	cloned.IdxCols = util.CloneColumns(op.IdxCols)
+	cloned.IdxColLens = make([]int, len(op.IdxColLens))
+	copy(cloned.IdxColLens, op.IdxColLens)
+	cloned.AccessConditions = util.CloneExpressions(op.AccessConditions)
+	cloned.ctx = newCtx
+	cloned.accessCols = util.CloneColumns(op.accessCols)
+	probeParents, ok := clonePhysicalPlansForPlanCache(newCtx, op.probeParents)
```

Comment:
we can move this a little bit up for quick fail over

Reply:
Updated. Thx.